### ISSUE TITLE
Transaction (for account) .stream never returns when passing the latest pagination token.

### ISSUE THREAD

**Issue by Paul776** [stellar/java-stellar-sdk#82]
Hello,

Not sure if this is by design or a bug but when using stream to listen to all the transaction for an account
  
```
TransactionsRequestBuilder transactionsRequest = server.transactions().forAccount(account).cursor(latestCursor);
transactionsWatcherSource = transactionsRequest.stream(new EventListener<TransactionResponse>() {

```

It blocks indefinitely until a transactions arrives and doesn't return directly (async). So if the address doesn't receive any transaction  the next 5 minutes the streams seems to be closing

Is that meant to be like this?

**Comment by jillesvangurp** [comment]
maybe related to #182

**Comment by tamirms** [comment]
@Paul776 are you able to reproduce this with the latest (version 0.7.0) release of the Java SDK? I just tried and `transactionsRequest.stream()` does not block for me

**Comment by jillesvangurp** [comment]
I changed the way streaming works a few months ago in #147 and also did a fix for #182. So, I think we can close this. If this is still a problem, we'll need a new issue with details.
